### ISSUE TITLE
Component: Micro Footer - Placeholder (Search by location or language)

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#3815]
## Detailed description

Issue : Foreground:#AFADAD
Background:#F4F4F4

The contrast ratio is: 2.0:1
Text failed at Level AA

Expected:
Provide below contrast ratio.
Foreground:#7F7E7E
Background:#F4F4F4

The contrast ratio is:3.7:1
Text passed at Level AA

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/92f22bf5-148f-46d1-aec6-af78f3874899)

**Comment by kennylam** [comment]
@Praveen-sr Where are you seeing this? The micro footer language-only search input looks like this
![image](https://user-images.githubusercontent.com/909118/93909639-f230c880-fccd-11ea-917e-98103f1521f9.png)

**Comment by kennylam** [comment]
Duplicate of https://github.com/carbon-design-system/ibm-dotcom-library/issues/3445
